### ISSUE TITLE
Example (+Test?) Checkpointing

### ISSUE THREAD

**Issue by robintibor** [braindecode/braindecode#237]
Skorch supplies us with checkpointing, might be worth to  integrate into existing tutorials/example ? or create a new example?

**Comment by hubertjb** [comment]
I agree, an example around checkpointing would be useful. For reference, there is checkpointing in the relative positioning example although it's not the main point of the example: https://braindecode.org/auto_examples/plot_relative_positioning.html#training
